core-api/src/main/java/com/optimizely/ab/odp/ODPManager.java

```diff
@@ -182,6 +212,8 @@ public ODPManager build() {
             if (eventManager == null) {
                 eventManager = new ODPEventManager(apiManager);
             }
+            eventManager.setUserCommonData(userCommonData != null ?  userCommonData : Collections.emptyMap());
```

Comment:
can we move this null check inside the setUserCommonData to ensure the null pointer exception won't occur inside odpEventManger line 137.